examples/fluentui-app/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
Is `web-vitals` important to mention? Is this generated from `react-scripts`?

Reply:
It's generated by the template, yep.

Reply:
I'm not sure about source controlling output from `react-scripts`. It could cause confusion like mine here. Are there any other options like scripting the template generation in the example? Might be a good unit test to make sure it stays working with "live" `react-scripts` rather than a snapshot of previous output.

Reply:
Oh, I probably didn't make myself clear last time. The `web-vitals` code is part of the FluentUI CRA template (see [here](https://github.com/microsoft/fluentui/tree/master/packages/cra-template/template/src)) and is not auto-generated by `react-scripts`, hence, as such, I don't think there is much we can or should do here. If the template itself removes it, then it will disappear from the example app here as well.

Reply:
I just ran `npx create-react-app my-app --template @fluentui/cra-template` and saw a lot of the same output that's here. It seems this code is autogenerated. Is there a reason we can't make `npx create-react-app my-app --template @fluentui/cra-template` part of the steps rather than controlling its output? The point being that this example should continue to work with whatever `npx create-react-app my-app --template @fluentui/cra-template` spits out, not a snapshot of its output at this point in time.

Reply:
Wait, but that's exactly what is happening here. The only thing we have to rewire is preventing minification which is required to be done regardless of the template used as long as `create-react-app` is used for bootstrapping.

Reply:
No, your understanding seems bang on. So my reasoning for including the contents of the `npx create-react-app` was so that first of all, all of the tweaks are applied immediately, and secondly, so that the potential user of `flamegrill` has some examples that effectively allow them to simply invoke `yarn`, `yarn build` and `yarn flamegrill` and explore the flamegraphs. This, in my opinion, lends itself well as a starting point of exploring the internals of flamegrill and flamegraphs in general, since it shields the user from the nitty-gritty details required just to get started. Let me know if that makes sense!